Linux Script/configure-linux.sh

```diff
@@ -384,10 +384,7 @@ getAuthToken() {
   if [ "$LOGGLY_AUTH_TOKEN" = "" ]; then
     logMsgToConfigSysLog "INFO" "INFO: Authentication token not provided. Trying to retrieve it from $LOGGLY_ACCOUNT_URL account."
     #get authentication token if user has not provided one
-    tokenstr=$(curl -s -u $LOGGLY_USERNAME:$LOGGLY_PASSWORD $LOGGLY_ACCOUNT_URL/apiv2/customer | grep -v "token")
-
-    #get the string from index 0 to first occurence of ,
-    tokenstr=${tokenstr%%,*}
+    tokenstr=$(curl -s -u $LOGGLY_USERNAME:$LOGGLY_PASSWORD $LOGGLY_ACCOUNT_URL/apiv2/customer | grep -A1 "token" | grep -v "token")
```

Comment:
Agree, this is independent to the position in json. 
I am just thinking why not to grep directly a token and remove the next 3 steps. Like:
grep -o -e '[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*-[a-f0-9]*'
Also this will work even when json comes unformatted (no new lines)

Reply:
Good idea, from my point of view it's a bit better to parse the json based on the attribute name, not based on its value, because if the json changed we could match something we do not want to.

I updated the parsing pattern to be more strict about the attribute name.